validator/starferry-upgrade.md

```diff
@@ -0,0 +1,202 @@
+---
+description: StarFerry is the third major upgrade to the LikeCoin chain
+---
+
+# StarFerry Upgrade
+
+### Please join the [#mainnet-validators](https://discord.gg/yGUqtcHGjv) channel.
+
+### Changelog
+
+- Upgrade cosmos-sdk to 0.45.6, ibc-go to 2.3.0
+- Introduce x/likenft module, and integrate x/nft module backported from cosmos-sdk 0.46.0-rc1
+
+### Timeline
+
+| Event                                                            | Time         |
+| ---------------------------------------------------------------- | ------------ |
+| Testnet Proposal                                                 | W4 June 2022 |
+| Testnet Deploy                                                   | W1 July 2022 |
+| [Mainnet Proposal](https://likecoin.bigdipper.live/proposals/46) | W1 July 2022 |
+| Mainnet Deploy                                                   | W2 July 2022 |
+
+### Chain Upgrades List
+
+| Upgrade Name | Binary Version | Binary Path                                   |
+| ------------ | -------------- | --------------------------------------------- |
+| FoTan        | v1.2.0         | `.liked/cosmovisor/genesis/bin/liked`         |
+| LaiChiKok    | v2.0.2         | `.liked/cosmovisor/upgrades/v2.0.0/bin/liked` |
+| StarFerry    | v3.0.0         | `.liked/cosmovisor/upgrades/v3.0.0/bin/liked` |
+
+### System Requirements During Upgrade
+
+The system resources required during the upgrade will be higher than normal operation for data migration tasks.
+
+In particular, large amount of memory will be used, and data files will be duplicated for backup.
+
+After the upgrade is applied, the day-to-day resources requirement is same as previously suggested in our Node Setup Guide.
+
+Below recommendation is based on recent test runs of migration with mainnet data. Any machine with less than 16GB will likely result in OOM (out of memory) error. Also note that using swap will lengthen the upgrade duration considerably.
+
+#### Recommended
+
+- 8 CPU Cores
+- 32GB RAM + 32GB Swap
+- 500GB+ SSD
+
+#### Minimum
+
+- 4 CPU Cores
+- 16GB RAM + 48GB Swap
```

Comment:
I think @hochiw comfirm < 32GB instance kill.

Reply:
it's without swap tho, RAM + Swap > 32gb should be okay

Reply:
arguably it will be super slow at 16 + 48 even if it doesn't crash... so maybe set minimum to 32+32?